pkg/daemons/control/server.go

```diff
@@ -421,20 +431,43 @@ func WritePasswords(passwdFile string, records [][]string) error {
 	return os.Rename(passwdFile+".tmp", passwdFile)
 }
 
+func genEncryptedNetworkInfo(controlConfig *config.Control, runtime *config.ControlRuntime) error {
```

Comment:
How can this be handled in HA mode? as far as I understand this function will result in creating a IPSECPSK for each master node